git-glossary.md

```diff
@@ -43,8 +51,8 @@ A collection of [references](#references) to commit objects and an [object datab
 
 ## references
 A string name that points to a commit. A reference also includes information about where the [commit](#commit) is located.
-For fork-clone style development the commit will be in one of three locations. 
+For the [fork/clone development workflow](fork-clone-workflow.md) the commit will be in one of three locations. 
 
 1. A local clone on a developer's computer
-1. The origin repository in the cloud
-1. The upstream repository in the cloud
+1. The origin repository in [github](#github), which is your remote [fork](#fork)
+1. The upstream repository in [github](#github), which is the organization's remote repository
```

Comment:
Do we want to mention the organization here? It's possible that the repo is held by an individual account right? I'm not sure what to call this though, "main repository"? Something else?

Reply:
Yes. I sometimes write the definition in git-glossary as if it was in the work flow section where we would want to use the term organization. Here, I'll change it to 'main repository'

Reply:
Or 'main remote repository'.